dbt/models/census/schema.yml

```diff
@@ -12,3 +12,135 @@ sources:
 
 models:
   - name: census.vw_acs5_stat
+    columns:
+      - name: percent_race_white
+        meta:
+          notes: Percentages will NOT sum to 1 since the Census allows more than 1 race per respondent
+      - name: percent_race_other
+        meta:
+          notes: Percentages will NOT sum to 1 since the Census allows more than 1 race per respondent
+      - name: percent_race_nhpi
+        meta:
+          notes: Percentages will NOT sum to 1 since the Census allows more than 1 race per respondent
+      - name: percent_race_hisp
+        meta:
+          notes: Counts for hispanic/latino population are complicated. See Census Reporter for more info
+      - name: percent_race_black
+        meta:
+          notes: Percentages will NOT sum to 1 since the Census allows more than 1 race per respondent
+      - name: percent_race_asian
+        meta:
+          notes: Percentages will NOT sum to 1 since the Census allows more than 1 race per respondent
+      - name: percent_race_aian
+        meta:
+          notes: Percentages will NOT sum to 1 since the Census allows more than 1 race per respondent
+      - name: percent_mobility_no_move
+        description: Percent of people (older than 1 year) who have not moved in the past 12 months
+        meta:
+          notes: ACS variable B07003_004E / B07003_001E
```

Comment:
One gripe that I have about this organizational scheme (summaries in `description`, details in `meta.notes`) is that when you click on a row in a dbt `Columns` table, the description is actually rendered _after_ the notes:

<img width="1011" alt="Screenshot 2023-09-08 at 3 32 29 PM" src="https://github.com/ccao-data/data-architecture/assets/14170650/735d8b0b-35d8-4784-b97d-f0ac5f27623c">

I'm not sure if there's anything we can do about this (e.g. swap the pattern so the summaries are in `meta.notes`) since the description is shown in truncated form in the table overview, and we probably want to maintain that behavior for the summaries. But I want to call it out as something I'm aware of and ambivalent about!